### ISSUE TITLE
cellDecorator missing from CheckboxRenderer

### ISSUE THREAD

**Issue by parasite83** [handsontable/handsontable#2242]
I could not add custom className via the cells option to a checkbox column. A possible fix this to add
```
Handsontable.renderers.cellDecorator.apply(this, arguments);
```
at the top of the CheckboxRenderer.

**Comment by AMBudnik** [comment]
related: https://github.com/handsontable/handsontable/issues/2925

**Comment by AMBudnik** [comment]
fixed with #3532 in Handsontable [0.25.1](https://docs.handsontable.com/0.25.1/tutorial-release-notes.html)
demo: http://jsfiddle.net/h4vwe3q5/
